sql/core/src/main/scala/org/apache/spark/sql/execution/BaseScriptTransformationExec.scala

```diff
@@ -119,7 +119,7 @@ trait BaseScriptTransformationExec extends UnaryExecNode {
         val kvWriter = CatalystTypeConverters.createToCatalystConverter(StringType)
         prevLine: String =>
           new GenericInternalRow(
-            prevLine.split(outputRowFormat).slice(0, 2)
+            prevLine.split(outputRowFormat).slice(0, 2).padTo(2, null)
```

Comment:
nit: could you leave comments about why we need `.padTo(2, null)` here.

Reply:
> nit: could you leave comments about why we need `.padTo(2, null)` here.

See the latest change, I have changed comment leaved before